app/scripts/views/force_auth.js

```diff
@@ -32,17 +37,35 @@ function (p, BaseView, FormView, SignInView, Template, Session) {
       this.user.clearSignedInAccount();
     },
 
-    context: function () {
-      var fatalError = '';
-      var email = this.relier.get('email');
+    beforeRender: function () {
+      var self = this;
+      var relier = self.relier;
+      var email = getEmail(relier);
+
       if (! email) {
-        fatalError = t('/force_auth requires an email');
+        self._fatalError = t('/force_auth requires an email');
```

Comment:
You know what would be more awesome? Convert this to an `AuthError` error with an `errno` that can be logged.